scripts/fund_e2e_account.sh

```diff
@@ -1,31 +0,0 @@
-#!/usr/bin/env sh
```

Comment:
we have a fund_e2e_accounts.sh file, and i think this one is not used anymore (couldn't find references to it)